From f84a4cb0df90098ffde4bd79ace5964d04497cfc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mateusz=20Miku=C5=82a?= <mati865@gmail.com>
Date: Fri, 14 May 2021 23:46:49 +0300
Subject: [PATCH] [LLD][MinGW] Ignore --no-undefined flag

AFAIK this is the default behaviour when this flag is not passed.

Differential Revision: https://reviews.llvm.org/D102516
---
 MinGW/Options.td | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MinGW/Options.td b/MinGW/Options.td
index 95b61952fc5e9..eceee3a4997b1 100644
--- a/MinGW/Options.td
+++ b/MinGW/Options.td
@@ -125,6 +125,7 @@ def: Flag<["--"], "full-shutdown">;
 def: F<"high-entropy-va">;
 def: S<"major-image-version">;
 def: S<"minor-image-version">;
+def: F<"no-undefined">;
 def: F<"nxcompat">;
 def: F<"pic-executable">;
 def: S<"plugin">;
